wordpress-mobile/gutenberg-mobile
# Fix makepot script using wp-i18n makepot

Address https://github.com/wordpress-mobile/gutenberg-mobile/issues/751

https://github.com/wordpress-mobile/gutenberg-mobile/pull/658 was a first attempt at fixing the mobile pot file generation.

Since then `babel-plugin-makepot` was removed from gutenberg and the official way is to use `wp i18n make-pot` instead. This PR updates our makepot script to use this.

### Testing Instructions
- Have `xgettext` installed
- Run `yarn makepot`
- Check that `./gutenberg.pot` and `gutenberg-mobile.pot` file were generated in the root folder
- `gutenberg-mobile.pot` should only contain the 4 following strings:
    - `ADD BLOCK HERE`
    - `Move block up`
    - `Move block down`
    - `Remove content`
All other strings in `./src` are already part of `gutenberg.pot`.

In a followup PR, we'll want to remove those 4 strings from this repository so all strings from this project are translated by the glopress gutenberg project
